### ISSUE TITLE
[Verified Importer] Failing for File's that have also content

### ISSUE THREAD

**Issue by FantasticoFox** [inblockio/DataAccounting#276]
As currently given in the code the importer is ether importing the file OR the content but is not combining it. In this case content of the page is lost leading to a wrong verification hash.

https://github.com/inblockio/DataAccounting/blob/45180b71cb1201e4f84baede3c4c127273297d58/includes/Transfer/Importer.php#L89-L93

See export json has both: filehash and comment of the page:

``` json
"content": {
                            "file_hash": "ed033051512699f74b930ea52143661ba94cc57b268b0f8fd8d822d4c922db02caf009744bc43421fe5cda17ea919e2203e6bf20f850e3b6fd0480cc140fe7f9",
                            "main": "Version 2 comment on file",
                            "transclusion-hashes": ""
                        },
 "content_hash": "25925805abca5181995f8d5d689b27c8c8c6a90ef1c56706a9f93f8ebd93cdb6388715ca7db50354777b0c7bbc928816e86787b793793c1a7f155f3dd9763c9a"
```
The content hash is calculated by file_hash + content_string. as the content_string is missing after upload it will lead to a failure.
![image](https://user-images.githubusercontent.com/45313235/148645297-bc10e54e-9b97-4bca-96b6-2f583c513a4f.png)

**Comment by FantasticoFox** [comment]
This effects also the content slots which are then not imported.

**Comment by FantasticoFox** [comment]
Acceptance criteria:

It is possible to without the page verification failing:
* Add additional data with 'edit' page works
* It is possible to witness and sign multiple revisions

**Comment by FantasticoFox** [comment]
![image](https://user-images.githubusercontent.com/45313235/148894354-d6cf40fd-813c-4bed-8063-c7422994e7bb.png)

FIXED in 943c9779ab5c08bb990f416c98e181dc36517521
